source/slang/hlsl.meta.slang

```diff
@@ -255,44 +255,84 @@ float __glsl_texture<TSampler, TCoord>(TSampler s, TCoord value)
 }
 
 __glsl_extension(GL_EXT_texture_shadow_lod)
-__target_intrinsic(glsl, "texture($0, $1)")
 [require(glsl, texture_shadowlod)]
-float __glsl_texture_1d_shadow<TSampler, TCoord>(TSampler s, TCoord value);
```

Comment:
do we want to ForceInline these functions?

Reply:
Functions whose body is a `__intrinsic_asm` is automatically inlined without needing [ForceInline].